pkg/influx/errors.go

```diff
@@ -15,6 +16,35 @@ import (
 	grpcstatus "google.golang.org/grpc/status"
 )
 
+// Error codes copied from https://github.com/influxdata/influxdb/blob/569e84d4a73250f2928136e5b40452d058a149bd/kit/platform/errors/errors.go#L15-L29
+const (
+	EInternal            = "internal error"
+	ENotImplemented      = "not implemented"
+	ENotFound            = "not found"
+	EConflict            = "conflict"             // action cannot be performed
+	EInvalid             = "invalid"              // validation failed
+	EUnprocessableEntity = "unprocessable entity" // data type is correct, but out of range
+	EEmptyValue          = "empty value"
+	EUnavailable         = "unavailable"
+	EForbidden           = "forbidden"
+	ETooManyRequests     = "too many requests"
+	EUnauthorized        = "unauthorized"
+	EMethodNotAllowed    = "method not allowed"
+	ETooLarge            = "request too large"
+)
```

Comment:
We might be able to get rid of these and import them if we update `github.com/influxdata/influxdb/v2` to the latest